### DISCUSSION TITLE
Is there any way to use pgBouncer and connection limit

### DISCUSSION THREAD

**Discussion by indahud** [supabase/supabase#775] [Questions]
I am looking to pass pgbouncer and connection limit to the database url. Is there any way to do that??

**Comment by steve-chavez** [comment]
@indahud What's your use case? The REST API already has a built-in pool.

**Reply by indahud** [reply]
@steve-chavez I am using supabase just for postgres database puropose. I am using prisma for my project. My site will open too many connections if the number of users increases. I would love to switch to supabase but that's lot of work for me. But in future I will definitely use supabase.

**Reply by steve-chavez** [reply]
@indahud Ah, I see. It's possible that we'll add pgbouncer in Supabase. We're having a discussion on https://github.com/supabase/postgres/pull/33, feel free to chime in.
